packages/cli-doctor/src/tools/healthchecks/xcodeEnv.ts

```diff
@@ -29,33 +31,37 @@ export default {
   getDiagnostics: async () => {
     const projectRoot = findProjectRoot();
     const allPathsHasXcodeEnvFile = findPodfilePaths(projectRoot)
-      .map((pathString) => {
+      .map((pathString: string) => {
         const basePath = removeLastPathComponent(pathString);
         return pathHasXcodeEnvFile(basePath);
       })
-      .reduce((previousValue, currentValue) => previousValue && currentValue);
+      .reduce(
+        (previousValue: boolean, currentValue: boolean) =>
+          previousValue && currentValue,
+      );
     return {
       needsToBeFixed: !allPathsHasXcodeEnvFile,
     };
   },
-  runAutomaticFix: async () => {
+  runAutomaticFix: async ({loader}) => {
+    loader.stop();
     const templateXcodeEnv = '_xcode.env';
     const projectRoot = findProjectRoot();
-
-    const templateIosPath = path.dirname(
-      require.resolve('react-native/template/ios'),
```

Comment:
if this silently failed, can we make sure the `runAutomaticFix` has a proper error handler that surfaces to the exit code?

Reply:
Added! 👍